### DISCUSSION TITLE
Reference

### DISCUSSION THREAD

**Discussion by Rahim-Habibi** [idaholab/moose#28940] [Q&A General]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

<Your question here> Hi all,
I would like to know more details about the theory you described here ([link](https://mooseframework.inl.gov/source/materials/CappedWeakPlaneStressUpdate.html)). Can you please refer me to paper or book you used for? 
Thanks

**Comment by loganharbour** [comment]
@sapitts looks like you were the one that wrote this; can you comment?

**Reply by sapitts** [reply]
Unfortunately all I did was to transfer this documentation over to MooseDocs from the old system. @WilkAndy is the one who wrote the original text. If he's not monitoring GitHub notifications, maybe @bwspenc can comment?

**Reply by Rahim-Habibi** [reply]
Hi @sapitts 
Thanks for your reply.
@bwspenc any comment? 
Rahim
